src/transform/expressionEvaluation.ml

```diff
@@ -129,7 +129,7 @@ module ExpEval : Transform.S =
         method private try_ask location expression =
           match ~? (fun () -> (ask location).Queries.f (Queries.EvalInt expression)) with
             (* Evaluable: Definite *)
-          | Some (`Lifted value) -> Some (Some (value <> Int64.zero))
+          | Some (`Lifted value) -> Some (Some (value <> IntOps.BigIntOps.zero))
```

Comment:
This comparison should probably delegate to `BigIntOps.equal` or whatever instead.

I suspect there may be similar issues around elsewhere as well.

Reply:
I only found this one case when some value is compared to an IntOps.BigIntOps. 
If I find more similar cases later, I'll change them too.